### PR TITLE
Improve performance for large scale MongoDB database

### PR DESCRIPTION
**Pull request by Ndpnt**

### CODE REVIEW

File: src/archivist/recorder/repositories/mongo/index.js [hunk 1]
```diff
@@ -24,6 +24,8 @@ export default class MongoRepository extends RepositoryInterface {
 
     this.collection = db.collection(this.collectionName);
 
+    await this.collection.createIndex({ serviceId: 1, termsType: 1, fetchDate: -1 });
```
Comment (MattiSG) [inline src/archivist/recorder/repositories/mongo/index.js:27]: Do we really want to block with `await`? 🤔

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,15 @@
 
 All changes that impact users of this module are documented in this file, in the [Common Changelog](https://common-changelog.org) format with some additional specifications defined in the CONTRIBUTING file. This codebase adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## Unreleased [patch]
+
+_Full changeset and discussions: [#1037](https://github.com/OpenTermsArchive/engine/pull/1037)._
+
+> Development of this release was made on a voluntary basis by [@Ndpnt](http://github.com/ndpnt).
```
Comment (MattiSG) [inline CHANGELOG.md:9]:
```suggestion
> Development of this release was made on a volunteer basis by [@Ndpnt](http://github.com/ndpnt).
```

**Review by MattiSG — APPROVED** [review]
LGTM! Not checked locally.

File: src/archivist/recorder/repositories/mongo/index.js (hunk 1 above)
Comment (Ndpnt) [reply src/archivist/recorder/repositories/mongo/index.js:27]: Not necessary

**Pull request merged**